<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>Publications</title>
<style>
h1 { 
    display: block;
    font-family: Georgia;
    font-size: 5em;
    margin-top: 0em;
    margin-bottom: 0em;
    margin-left: 0;
    margin-right: 0;
}
h2 {
    display: block;
    font-size: 1.5em;
    margin-top: 0.83em;
    margin-bottom: 0.83em;
    margin-left: 0;
    margin-right: 0;
    font-weight: bold;
    color: silver;
}
h5 { 
    display: block;
    font-size: 0.9em;
    margin-top: 1.67em;
    margin-bottom: 1.67em;
    margin-left: 0;
    margin-right: 0;
    font-weight: bold;
    color: black;
}
a:link {
    color: white;
    text-decoration: none;
    font-weight: bold;
}
a:visited {
    color: white;
    text-decoration: none;
    font-weight: bold;
}
a:hover {
    color: white;
    text-decoration: none;
    font-weight: bolder;
}

.box {
    width:800px;
    height:200px;
    position:fixed;
    margin-left:-400px; /* half of width */
    margin-top:-225px;  /* half of height */
    left:50%;
    top:40%;
}

table {
  table-layout: fixed ;
  width: 150% ;
}

td {
  width: 25% ;
}

.big {
  font-size: 1.0em;
}

.small {
  font-size: .7em;
}

.square {
  width: .7em;
  height: .7em;
  margin: .5em;
  display: inline-block;
}

/* Custom dropdown */
.custom-dropdown {
  position: relative;
  display: inline-block;
  vertical-align: middle;
  margin: 10px; /* demo only */
}

.custom-dropdown select {
  background-color: #ffffff80; /* #2c3e50; */
  color: #000;
  font-size: inherit;
  padding: .5em;
  padding-right: 2.5em; 
  border: 1;
  margin: 0;
  border-radius: 3px;
  text-indent: 0.01px;
  text-overflow: '';
  -webkit-appearance: none; /* hide default arrow in chrome OSX */
}

.custom-dropdown::before,
.custom-dropdown::after {
  content: "";
  position: absolute;
  pointer-events: none;
}

.custom-dropdown::after { /*  Custom dropdown arrow */
  content: "\25BC";
  height: 1em;
  font-size: .625em;
  line-height: 1;
  right: 1.2em;
  top: 50%;
  margin-top: -.5em;
}

.custom-dropdown::before { /*  Custom dropdown arrow cover */
  width: 2em;
  right: 0;
  top: 0;
  bottom: 0;
  border-radius: 0 3px 3px 0;
}

.custom-dropdown select[disabled] {
  color: rgba(0,0,0,.3);
}

.custom-dropdown select[disabled]::after {
  color: rgba(0,0,0,.1);
}

.custom-dropdown::before {
  background-color: rgba(0,0,0,.15);
}

.custom-dropdown::after {
  color: rgba(0,0,0,.4);
}

.strokeme
{
    color: black;
    text-shadow:
    -1px -1px 0 #000,
    1px -1px 0 #000,
    -1px 1px 0 #000,
    1px 1px 0 #000;
}

</style>
</head>
<body style="background: url(hoodoo.jpeg) no-repeat center fixed;background-size: cover;width: 50%;margin: auto;">
<table class="box" align=center>
<tr><td align=center colspan=4><H1 class="strokeme"><a href=https://warrenlr.github.io title="Hoodoo Glitter, Writing-on-Stone Provincial Park, AB (Photo: RL Warren)">RL Warren</a></H1></td></tr>
<tr><td align=center>
<span class="custom-dropdown">
<SELECT onChange='top.location.href=this.options[selectedIndex].value;' Size='1' WIDTH='192' STYLE='width: 192px'>
<OPTION VALUE=https://warrenlr.github.io/papers/>&nbsp;&nbsp;&nbsp;&nbsp;ACCOLADES</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/science_in_the_city_7may2020.pdf>[2020] Featured in Science In The City for ntJoin development</OPTION> 
<OPTION VALUE=http://scholar.google.ca/citations?hl=en&user=nLkZYtcAAAAJ&view_op=list_works&is_public_preview=1>[2016] SSAKE manuscript cited over 675 times</OPTION>
<OPTION VALUE=https://bccancerfoundation.com/about-us/news/jambor-mccarthy-legacy-and-bc-cancer-foundation-make-bc-philanthropic-history-214>[2015|16|21] John Jambor knowledge fund recipient</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/Warren_SBN2015_final.pdf>[2015] Speaker/Mentor at BC Student Biotech Network SBN</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/fuso_TIMEtop10_2011.pdf>[2011] Colon cancer-Fusobacterium discovery on TIME magazine's top ten medical breakthroughs of the year</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/GeorgiaStraightFusoCRC_nov2011.pdf>[2011] Interviewed with newspaper for Fusobacterium discovery</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ntn24_rlw-interview_oct2011.mp4>[2011] Interviewed with TV channel for Fusobacterium discovery</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/iSSAKE_The_Scientist_Jun09.pdf>[2009] Interviewed with The Scientist for work on profiling T cell metagenomes</OPTION>
<OPTION VALUE=http://packages.debian.org/ssake>[2009] SSAKE packaged in the Debian operating system</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/bioinfo_editorial09.pdf>[2009] SSAKE short-listed in the editorial of the scientific journal Bioinformatics</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/GTechSeqGuide1208.pdf>[2008] One of 6 experts invited to discuss NGS bioinformatics</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/technote_thirdparty_software.pdf>[2006|14] SSAKE one of four 3rd-party genomics application recommended by Illumina Ltd.</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/amia08.pdf>[2008] SSAKE recognized by Russ Altman, MD PhD as one of 27 publication with most impact</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/bioinform07.pdf>[2007] Interviewed by GenomeWeb for the release of SSAKE</OPTION>
<OPTION VALUE=http://warrenlr.github.io/papers>[2002|04] Interviewed with french CBC about the GSC and bovine genome release</OPTION>
<OPTION VALUE=http://www.f1000biology.com/article/id/1023410>[2004] 'faculty of 1000' recognizes article "Warren et al. 2004. Functional characterization.."</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/gene-expression-system.pdf>[2001] Co-engineered a gene switch (BRI-NRC) - work patented, sold to QBIOgene</OPTION>
<OPTION VALUE=http://warrenlr.github.io/papers>[1999] Awarded FCAR-FRSQ Ph.D. Scholarship - declined</OPTION>
<OPTION VALUE=http://www.grad.ubc.ca/awards/index.asp?menu=004,000,000,000>[1998] Awarded University of British Columbia Graduate Fellowship (UGF)  </OPTION>
<OPTION VALUE=http://www.frqs.gouv.qc.ca/>[1997] Recipient of excellence award from the FRSQ for Honor's thesis</OPTION>
<OPTION VALUE=http://www.frqs.gouv.qc.ca/>[1996] Scholarship recipient from Fonds de la recherche en sante Quebec (FRSQ)</OPTION>
<OPTION VALUE=http://www.nasa.gov/mission_pages/shuttle/shuttlemissions/archives/sts-69.html>[1995] Worked at NASA with the purpose of crystallizing proteins under microgravity</OPTION>
</SELECT>
</span>
</td>
<td align=center>
<span class="custom-dropdown">
<SELECT onChange='top.location.href=this.options[selectedIndex].value;' Size='1' WIDTH='192' STYLE='width: 192px'>
<OPTION VALUE=https://warrenlr.github.io/papers/>&nbsp;&nbsp;&nbsp;SCIENCE+ART</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/xmv-hive_i0_b1_c42_a0.9.svg>Coronavirus genome synteny</OPTION>
<OPTION VALUE=https://bcgsc.github.io/SARS2>SARS-CoV-2 mutation timemaps</OPTION>
</SELECT>
</span>
</td>
<td align=center>
<span class="custom-dropdown">
<SELECT onChange='top.location.href=this.options[selectedIndex].value;' Size='1' WIDTH='192' STYLE='width: 192px'>
<OPTION VALUE=https://warrenlr.github.io/papers/>&nbsp;&nbsp;PUBLICATIONS</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2022 ---</OPTION>
<OPTION VALUE=https://doi.org/10.1002/cpz1.442>[article] ntEdit+Sealer: targeted automated finishing of long-read genome assemblies</OPTION>
<OPTION VALUE=https://doi.org/10.1186/s12864-022-08310-4>[article] AMPlify: attentive deep learning model for discovery of novel antimicrobial peptides</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2021 ---</OPTION>
<OPTION VALUE=https://doi.org/10.1093/nargab/lqab105>[article] RNA-Scoop: interactive visualization of transcripts in single-cell transcriptomes</OPTION>
<OPTION VALUE=https://doi.org/10.1186/s12859-021-04451-7>[article] LongStitch: high-quality genome assembly correction and scaffolding using long reads</OPTION>
<OPTION VALUE=http://doi.org/10.7717/peerj.12368>[article] HLA alleles measured from COVID-19 patient transcriptomes reveal associations with disease prognosis in a New York cohort</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2021_GapPredict_IEEETCBB.pdf>[article] GapPredict: A Language Model for Resolving Gaps in Draft Genome Assemblies</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2021_WarrenRene_SARS-CoV-2_variantTimemaps_ISMBposterBioVis-COVID19.pdf>[poster] ISMB, Virtual - SARS-CoV-2 variant timemaps</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2021_WarrenRene_SARS-CoV-2_variantTimemaps_ISMBtalkBioVis-COVID19.pdf>[seminar] ISMB, Virtual - SARS-CoV-2 variant timemaps</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2021_SARScoV2vartimemaps_F1000.pdf>[article] Interactive SARS-CoV-2 mutation timemaps</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2021_SARS2mutations_arxiv.pdf>[preprint] Interactive SARS-CoV-2 mutation timemaps</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2020 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_COVID19HLA_medrxiv.pdf>[preprint] COVID-19 patients HLA markers of disease prognosis</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_COVID19HLA_bioinformatics.pdf>[article] HLA predictions from COVID-19 patients at the pandemic onset</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_blackSpruceChloroplast_MRA.pdf>[article] Complete chloroplast genome sequence of a black spruce from eastern Canada</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_RNAbloom_genomeRes.pdf>[article] RNA-Bloom: Single-cell transcriptome assembler</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_Trans-NanoSim_GigaSci.pdf>[article] Trans-NanoSim characterizes and simulates nanopore RNA-seq data</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_sitkaSpruceMitochondrialGenome_GBE.pdf>[article] Complete mitochondrial genome of a gymnosperm, Sitka spruce</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_APR1_COVID-19-BAL-HLA_preprints.pdf>[preprint] HLA predictions in COVID-19 patients at the pandemic onset</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_ntjoin_bioinformatics.pdf>[article] ntJoin: Fast and lightweight assembly-guided scaffolding using minimizer graphs</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2020_ehap_genomics.pdf>[article] eHAP1 human haploid cell line hybrid reference genome from short/long reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2019 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ntedit_ismb2019.pdf>[poster] ISMB, Basel - ntEdit: ultra fast, scalable genome sequence polishing</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2019_engelmannSpruceChloroplast_MRA.pdf>[article] Complete chloroplast genome sequence of an Engelmann spruce from western Canada</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2019_whiteSpruceChloroplast_MRA.pdf>[article] Complete chloroplast genome sequence of a white spruce from eastern Canada</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2019_ntedit_bioinformatics.pdf>[article] ntEdit: scalable genome sequence polishing</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2019_AMP_NatureSciReports.pdf>[article] Antimicrobial peptides from Rana [Lithobates] catesbeiana: Gene structure and bioinformatic identification of novel forms from tadpoles</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/DeNovoAssemblyBTL.pdf>[resource] Scalable de novo genome assembly </OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2018 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_tigmint_BMCbioinfo.pdf>[article] Tigmint: correcting assembly errors using linked reads from large molecules</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_APA_BMCgenomics.pdf>[article] Recurrent tumor-specific regulation of alternative polyadenylation of cancer-related genes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_arks_bmcbioinfo.pdf>[article] ARKS: chromosome-scale scaffolding of human genome drafts with linked read kmers</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_ARKS_RECOMB.pdf>[poster] RECOMB, Paris - Chromosome-scale scaffolding human genomes with linked read kmers</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_xmatchview_joss.pdf>[article] Visualizing genome synteny with xmatchview</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_chopstitch_bioinformatics.pdf>[article] ChopStitch: exon annotation and splice graph construction</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/GenomeAssemblyToolkitBTL.pdf>[resource] Genome Assembly Toolkit</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2017 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_seaotter_genes.pdf>[article] The Genome of the Northern Sea Otter</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_beluga_genes.pdf>[article] The Genome of the Beluga Whale</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_bullfrog_natcomm.pdf>[article] The North American bullfrog draft genome provides insight into hormonal regulation of long noncoding RNA</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_abyss2_gr.pdf>[article] ABySS 2.0: Resource-efficient assembly of large genomes using a Bloom filter</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_mycobacterium_ga.pdf>[article] Complete genome sequence of Mycobacterium chimaera SJ42</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_ARCS_bioinformatics.pdf>[article] ARCS: Scaffolding genome drafts with linked reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_arcs_recombseq.pdf>[seminar] RECOMB-Seq, Hong Kong - Scaffolding with linked reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_nanosim_GS.pdf>[article] NanoSim: nanopore sequence read simulator based on statistical characterization</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_kollector_bioinformatics.pdf>[article] Kollector: transcript-informed, targeted de novo assembly of gene loci</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_overlap_bioinformatics.pdf>[article] Innovations and challenges in detecting long read overlaps</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2016 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_RAILS_joss.pdf>[article] RAILS and Cobbler: Scaffolding and automated finishing of draft genomes using long DNA sequences</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_hlaminer_ashg.pdf>[poster] ASHG, Vancouver Canada - HLAminer</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_Sitka_PO.PDF>[article] Assembly of the complete Sitka spruce chloroplast genome using 10x Genomics GemCode sequencing data</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_xlf_blood.pdf>[article] XLF deficiency affects TCR V(D)J recombination</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/WARREN_ISMB2016.pdf>[seminar] ISMB2016, Orlando, Florida USA - Genome assembly toolkit</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2015 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_PG29organelles_GBE.pdf>[article] Organellar genomes of white spruce (Picea glauca): assembly and annotation</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_konnector2_BMG.pdf>[article] Konnector v2.0: pseudo-long reads from paired-end sequencing data</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_Spruce_PJ.pdf>[article] Improved white spruce (Picea glauca) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_LINKS_GS.pdf>[article] LINKS: Scalable, alignment-free scaffolding of draft genomes with long reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_Sealer_BMC.pdf>[article] Sealer: a scalable gap-closing application for finishing draft genomes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_Spacedseed_IJG.pdf>[article] Spaced seed data structures for de novo assembly</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_EVADR_GM.pdf>[article] Activation of an endogenous retrovirus-associated long non-coding RNA in human adenocarcinoma</OPTION> 
<OPTION VALUE=https://warrenlr.github.io/papers/LINKSismb2015final.pdf>[seminar] ISMB2015, Dublin, Ireland - Scaffolding genome assemblies with nanopore reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/Warren_et_al_PSB2015Poster.pdf>[poster] Pacific Symposium on Biocomputing, Hawaii, USA - High-performance computing for assembly and analysis of big genomics data </OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_IgKL_genesimmunity.pdf>[article] Sequencing the human IG light chain</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2014 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2014_konnector_BIBM.pdf>[article] Konnector: Connecting paired-end reads using a Bloom filter de Bruijn graph</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2014_spacedseed_BIBM.pdf>[article] Spaced seed data structures</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2014_neoantigens_genomeres.pdf>[article] Neo-antigens from tumor genome meta-analysis correlate with survival</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2013 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2013_fusoCRC_microbiome.pdf>[article] Co-occurrence of anaerobic bacteria in colorectal carcinomas</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2013_brainhiv_plos.pdf>[article] Brain microbial populations in HIV/AIDS: Î±-proteobacteria predominate independent of host immune status</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2013_IgH_AJHG.pdf>[article] Complete haplotype sequence of human IgH</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2012 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2012_hlaminer_GM.pdf>[article] Derivation of HLA types from shotgun sequence datasets</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/Warren_HPTASR_PSB2012.pdf>[poster] Pacific Symposium on Biocomputing, Hawaii, USA - Computational HLA prediction</OPTION>
<OPTION VALUE=https://google.com/patents/WO2012045150A1?cl=ar>[patent] Fusobacterium for colon cancer diagnosis</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2011 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_fusobacterium_genomeres.pdf>[article] Fusobacterium infection is prevalent in human colorectal carcinoma</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_IAdetection_plos.pdf>[article] Identifying microbes with short reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_TASR_plos.pdf>[article] TASR: targeted assembly of sequence reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_hervbrain_plos.pdf>[article] HERV expression in brain</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_TCRB_genomeres.pdf>[article] Exhaustive T cell repertoire sequencing</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_crypto_mbio.pdf>[article] Genome variation in Cryptococcus gattii</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2010 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/santafe2010.pdf>[seminar] SFAF2010, Santa Fe, USA - Sequencing human IgH from H.mole</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2010_epitopecensus_humimm.pdf>[article] A census of predicted mutational epitopes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2009 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_breastcancer_nature.pdf>[article] Mutational evolution in a lobular breast tumour</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_TCRBseq_genomeres.pdf>[article] Profiling the T-cell receptor beta-chain</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_SCODA_PNAS.pdf>[article] Unique parameter for separation of biomolecules</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/tcrb4x4_agbt09.pdf>[poster] AGBT, Florida, USA - Profiling model TCRb</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ssake4x3_agbt09.pdf>[poster] AGBT, Florida, USA - SSAKE 3.2.1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_iSSAKE_bioinformatics.pdf>[article] Profiling model T cell metagenomes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2008 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2008_synbio_genomeres.pdf>[article] Transcription of foreign DNA in Escherichia coli</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2008_clonerecomb_synbio.pdf>[article] Constructing large DNA segments by clone recombination</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ssake2008.pdf>[poster] Pacific Symposium on Biocomputing, Hawaii, USA - SSAKE 3.0</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2007 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/zurich2007.pdf>[seminar] Synthetic Biology 3.0, Zurich, Switzerland -Genome reconstruction</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2007_rebuildinggenome_bioessays.pdf>[article] Rebuilding microbial genomes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2006 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_SSAKE_bioinformatics.pdf>[article] Assembling millions of short DNA sequences using SSAKE</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_cumate_bmcbiotech.pdf>[article] The cumate gene-switch</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_lncap_bmcgenomics.pdf>[article] Analysis of the prostate cancer cell line LNCaP transcriptome</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_thor_bioinformatics.pdf>[article] THOR: Targeted high-throughput ortholog reconstructor</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_rha_pnas.pdf>[article] The complete genome of Rhodococcus sp. RHA1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_mapwgs_genomeres.pdf>[article] Physical map-assisted WGSS assemblies</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_uhordei_fungalgen.pdf>[article] Mating factor linkage and genome evolution in pathogens of cereals</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2005 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2005_sam_biotech.pdf>[article] Management of WGS assemblies using SAM</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/fassi2005.pdf>[poster] Genome Informatics, CSHL NY</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2004 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2004_prhl3_jbact.pdf>[article] Characterization of a catabolic plasmid from Rhodococcus RHA1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/rhava2004.pdf>[poster] Computational Genomics, VA USA - Rhodococcus WGA</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/rhauk2004.pdf>[poster] Genomes 2004, Hinxton UK - WGA of Rhodococcus sp. RHA1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- 2003 ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/sam2003.pdf>[poster] Genome Informatics, CSHL NY</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/>--- Eons ---</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cng2002.html>[article] Regulation of the rod photoreceptor cGMP channel</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cng2000.html>[article] Biochemical analysis of cGMP channels of rods</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cng1999.html>[article] cGMP channels and peripherin complex of rod cells</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/1999_WarrenRL_MScThesis_UBC.pdf>[thesis] Insights on the regulation of the rod cgmp-gated channel in vision</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/1997_nkx_emboj.pdf>[article] The cardiac transcription factors Nkx2-5+GATA-4 are cofactors</OPTION>
</SELECT>
</span>
</td>
<td align=center>
<span class="custom-dropdown">
<SELECT onChange='top.location.href=this.options[selectedIndex].value;' Size='1' WIDTH='192' STYLE='width: 192px'>
<OPTION VALUE=https://warrenlr.github.io/papers/>&nbsp;&nbsp;&nbsp;PHOTOGRAPHY</OPTION>
<OPTION VALUE=https://photos.app.goo.gl/ZHVgzYrRATHh7Q7j9>Stars and Skies</OPTION>
<OPTION VALUE=https://goo.gl/photos/X3mhTNGqDz6s62y57>Nature album</OPTION>
<OPTION VALUE=https://warrenlr.github.io/poems2021_venturingintounknown>Poems | Venturing into the unknown</OPTION>
<OPTION VALUE=https://warrenlr.github.io/yukon2020_photojourney>Yukon | A photographic journey</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cosmos2019.pdf>Cosmos | Journey into ethereal beauty</OPTION>
</SELECT>
</span>
</td>
</tr>
</table>
</body></html>
