<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>Publications</title>
</head><body style="background-image:url(iarm_rlw2017.JPG), height: 200px;">
<table align=center>
<tr><td>&nbsp;</td></tr>
<tr><td align=left><H1><a href=http://renewarren.ca>RenéWarren.ca</a></H1></td></tr>
<tr><td>&nbsp;</td></tr>
<tr><td align=center>
<H2>ACCOLADES</H2>
<SELECT onChange='top.location.href=this.options[selectedIndex].value;' Size='1' WIDTH='190' STYLE='width: 190px'>
<OPTION VALUE=https://warrenlr.github.io/papers/>Select</OPTION>
<OPTION VALUE=http://scholar.google.ca/citations?hl=en&user=nLkZYtcAAAAJ&view_op=list_works&is_public_preview=1>2016 - SSAKE manuscript cited over 500 times</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/Warren_SBN2015_final.pdf>2015 - Speaker/Mentor at BC Student Biotech Network SBN</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/fuso_TIMEtop10_2011.pdf>2011 - Fuso discovery on TIME magazine's top ten medical breakthroughs of the year</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/GeorgiaStraightFusoCRC_nov2011.pdf>2011 - Interviewed with newspaper for Fusobacterium discovery</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ntn24_rlw-interview_oct2011.mp4>2011 - Interviewed with TV channel for Fusobacterium discovery</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/iSSAKE_The_Scientist_Jun09.pdf>2009 - Interviewed with The Scientist for work on profiling T cell metagenomes</OPTION>
<OPTION VALUE=http://packages.debian.org/ssake>2009 - SSAKE packaged in the Debian operating system</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/bioinfo_editorial09.pdf>2009 - SSAKE short-listed in the editorial of the scientific journal Bioinformatics</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/GTechSeqGuide1208.pdf>2008 - One of 6 experts invited to discuss NGS bioinformatics</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/technote_thirdparty_software.pdf>2006|14 - SSAKE one of four 3rd-party genomics application recommended by Illumina Ltd.</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/amia08.pdf>2008 - SSAKE recognized by Russ Altman, MD PhD as one of 27 publication with most impact</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/bioinform07.pdf>2007 - Interviewed by GenomeWeb for the release of SSAKE</OPTION>
<OPTION VALUE=http://renewarren.ca>2002|04 - Interviewed with french CBC about the GSC and bovine genome release</OPTION>
<OPTION VALUE=http://www.f1000biology.com/article/id/1023410>2004 -  'faculty of 1000' recognizes paper "Warren et al. 2004. Functional characterization.."</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/gene-expression-system.pdf>2001 - Engineered a gene switch (BRI-NRC) - work was patented, sold to QBIOgene</OPTION>
<OPTION VALUE=http://www.grad.ubc.ca/awards/index.asp?menu=004,000,000,000>1998 - Awarded University of British Columbia Graduate Fellowship (UGF)  </OPTION>
<OPTION VALUE=http://www.frqs.gouv.qc.ca/>1997 - Recipient of excellence award from the FRSQ for Honor's thesis</OPTION>
<OPTION VALUE=http://www.frqs.gouv.qc.ca/>1996 - Scholarship recipient from Fonds de la recherche en sante Quebec (FRSQ)</OPTION>
<OPTION VALUE=http://www.nasa.gov/mission_pages/shuttle/shuttlemissions/archives/sts-69.html>1995 - Worked at NASA with the purpose of crystallizing proteins under microgravity</OPTION>
</SELECT>
<H2>PUBLICATIONS</H2>
<SELECT onChange='top.location.href=this.options[selectedIndex].value;' Size='1' WIDTH='190' STYLE='width: 190px'>
<OPTION VALUE=https://warrenlr.github.io/papers/>Select</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_chopstitch_bioinformatics.pdf>2018 -paper- ChopStitch: exon annotation and splice graph
construction</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2018_xmatchview_joss.pdf>2018 -paper- Visualizing genome synteny with xmatchview</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_seaotter_genes.pdf>2017 -paper- The Genome of the Northern Sea Otter</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_beluga_genes.pdf>2017 -paper- The Genome of the Beluga Whale</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_bullfrog_natcomm.pdf>2017 -paper- The North American bullfrog draft genome provides insight into hormonal regulation of long noncoding RNA</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_mycobacterium_ga.pdf>2017 -paper- Complete Genome Sequence of Mycobacterium chimaera SJ42</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_ARCS_bioinformatics.pdf>2017 -paper- ARCS: Scaffolding Genome Drafts with Linked Reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_arcs_recombseq.pdf>2017 -talk- RECOMB-Seq, Hong Kong - Scaffolding with linked reads 
<OPTION VALUE=https://warrenlr.github.io/papers/2017_nanosim_GS.pdf>2017 -paper- NanoSim: nanopore sequence read simulator based on statistical characterization</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_kollector_bioinformatics.pdf>2017 -paper- Kollector: transcript-informed, targeted de novo assembly of gene loci</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_overlap_bioinformatics.pdf>2017 -paper- Innovations and challenges in detecting long read overlaps</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_RAILS_joss.pdf>2016 -paper- RAILS and Cobbler: Scaffolding and automated finishing of draft genomes using long DNA sequences</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2017_abyss2_gr.pdf>2016 -paper- ABySS 2.0: Resource-Efficient Assembly of Large Genomes using a Bloom Filter</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_Sitka_PO.PDF>2016 -paper- Assembly of the Complete Sitka Spruce Chloroplast Genome Using 10X Genomics GemCode Sequencing Data</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2016_xlf_blood.pdf>2016 -paper- XLF deficiency affects TCR V(D)J recombination</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/WARREN_ISMB2016.pdf>2016 -talk- ISMB2016, Orlando, Florida USA - Genome Assembly Toolkit</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_PG29organelles_GBE.pdf>2015 -paper- Organellar Genomes of White Spruce (Picea glauca): Assembly and Annotation</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_konnector2_BMG.pdf>2015 -paper- Konnector v2.0: pseudo-long reads from paired-end sequencing data</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_Spruce_PJ.pdf>2015 -paper- Improved white spruce (Picea glauca) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_LINKS_GS.pdf>2015 -paper- LINKS: Scalable, alignment-free scaffolding of draft genomes with long reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_Sealer_BMC.pdf>2015 -paper- Sealer: a scalable gap-closing application for finishing draft genomes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_Spaceseed_IJG.pdf>2015 -paper- Spaced Seed Data Structures for De Novo Assembly</OPTIONS>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_EVADR_GM.pdf>2015 -paper- Activation of an endogenous retrovirus-associated long non-coding RNA in human adenocarcinoma</OPTION> 
<OPTION VALUE=https://warrenlr.github.io/papers/LINKSismb2015final.pdf>2015 -talk- ISMB2015, Dublin, Ireland - Scaffolding Genome Assemblies with Nanopore Reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/Warren_et_al_PSB2015Poster.pdf>2015 -meeting- Pacific Symposium on Biocomputing, Hawaii, USA - High-Performance Computing for Assembly and Analysis of Big Genomics Data </OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2015_IgKL_genesimmunity.pdf>2015 -paper- Sequencing the human IG light chain</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2014_konnector_BIBM.pdf>2014 -paper- Konnector: Connecting paired-end reads using a Bloom filter de Bruijn graph</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2014_spacedseed_BIBM.pdf>2014 -paper- Spaced seed data structures</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2014_neoantigens_genomeres.pdf>2014 -paper- Neo-antigens from tumor genome meta-analysis correlate with survival</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2013_fusoCRC_microbiome.pdf>2013 -paper- Co-occurrence of anaerobic bacteria in colorectal carcinomas</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2013_brainhiv_plos.pdf>2013 -paper- Brain microbial populations in HIV/AIDS: α-proteobacteria predominate independent of host immune status</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2013_IgH_AJHG.pdf>2013 -paper- Complete haplotype sequence of human IgH</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2012_hlaminer_GM.pdf>2012 -paper- Derivation of HLA types from shotgun sequence datasets</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/Warren_HPTASR_PSB2012.pdf>2012 -meeting- Pacific Symposium on Biocomputing, Hawaii, USA - Computational HLA prediction</OPTION>
<OPTION VALUE=https://google.com/patents/WO2012045150A1?cl=ar>2012 -patent- Fusobacterium for colon cancer diagnosis</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_fusobacterium_genomeres.pdf>2012 -paper- Fusobacterium infection is prevalent in human colorectal carcinoma</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_IAdetection_plos.pdf>2011 -paper- Identifying microbes with short reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_TASR_plos.pdf>2011 -paper- Targeted Assembly of Sequence Reads</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_hervbrain_plos.pdf>2011 -paper- HERV expression in brain</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_TCRB_genomeres.pdf>2011 -paper- Exhaustive T cell Repertoire Sequencing</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2011_crypto_mbio.pdf>2011 -paper- Genome Variation in Cryptococcus gattii</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/santafe2010.pdf>2010 -talk- SFAF2010, Santa Fe, USA - Sequencing human IgH from H.mole</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2010_epitopecensus_humimm.pdf>2010 -paper- A census of predicted mutational epitopes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_breastcancer_nature.pdf>2009 -paper- Mutational evolution in a lobular breast tumour</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_TCRBseq_genomeres.pdf>2009 -paper- Profiling the T-cell receptor beta-chain</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_SCODA_PNAS.pdf>2009 -paper- Unique parameter for separation of biomolecules</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/tcrb4x4_agbt09.pdf>2009 -meeting- AGBT, Florida, USA - Profiling model TCRb</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ssake4x3_agbt09.pdf>2009 -meeting- AGBT, Florida, USA - SSAKE 3.2.1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2009_iSSAKE_bioinformatics.pdf>2009 -paper- Profiling model T cell metagenomes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2008_synbio_genomeres.pdf>2008 -paper- Transcription of foreign DNA in Escherichia coli</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2008_clonerecomb_synbio.pdf>2008 -paper- Constructing large DNA segments by clone recombination</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/ssake2008.pdf>2008 -meeting- Pacific Symposium on Biocomputing, Hawaii, USA - SSAKE 3.0</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/zurich2007.pdf>2007 -talk- Synthetic Biology 3.0, Zurich, Switzerland -Genome reconstruction</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2007_rebuildinggenome_bioessays.pdf>2007 -paper- Rebuilding microbial genomes</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_SSAKE_bioinformatics.pdf>2007 -paper- Assembling millions of short DNA sequences using SSAKE</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_cumate_bmcbiotech.pdf>2006 -paper- The Cumate gene-switch</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_lncap_bmcgenomics.pdf>2006 -paper- Analysis of the prostate cancer cell line LNCaP transcriptome</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_thor_bioinformatics.pdf>2006 -paper- THOR: Targeted high-throughput ortholog reconstructor</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_rha_pnas.pdf>2006 -paper- The complete genome of Rhodococcus sp. RHA1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_mapwgs_genomeres.pdf>2006 -paper- Physical map-assisted WGSS assemblies</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2006_uhordei_fungalgen.pdf>2006 -paper- Mating factor linkage and genome evolution in pathogens of cereals</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2005_sam_biotech.pdf>2005 -paper- Management of WGSS assemblies using SAM</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/fassi2005.pdf>2005 -meeting- Genome Informatics, CSHL NY</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/2004_prhl3_jbact.pdf>2004 -paper- Characterization of a catabolic plasmid from Rhodococcus RHA1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/rhava2004.pdf>2004 -meeting- Computational Genomics, VA USA - Rhodococcus WGA</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/rhauk2004.pdf>2004 -meeting- Genomes 2004, Hinxton UK - WGA of Rhodococcus sp. RHA1</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/sam2003.pdf>2003 -meeting- Genome Informatics, CSHL NY</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cng2002.html>2002 -paper- Regulation of the rod photoreceptor cGMP channel</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cng2000.html>2000 -paper- Biochemical analysis of cGMP channels of rods</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/cng1999.html>1999 -paper- cGMP channels and peripherin complex of rod cells</OPTION>
<OPTION VALUE=https://warrenlr.github.io/papers/1997_nkx_emboj.pdf>1997 -paper- The cardiac transcription factors Nkx2-5+GATA-4 are cofactors</OPTION>
</SELECT>
</td>
<tr><td>&nbsp;</td></tr>
<tr><td>&nbsp;</td></tr>
<tr><td>&nbsp;</td></tr>
<tr><td>&nbsp;</td></tr>
<tr><td align=center>(c) RLW 2002-2018</td></tr>
</table>
</body></html>
